docs/intentReaderHandlers/HealthCheckHandler.md

```diff
@@ -0,0 +1,110 @@
+# HealthCheckHandler 
+
```

Comment:
what does point 2 mean ?
2. Confirms the healthCheck status.
also what does this mean ? This handler is responsible for sending the healthCheck command and validating whether FCA is launched as a third-party app or not.

Reply:
updated